constraints.txt

```diff
@@ -0,0 +1,2 @@
+git+git://github.com/david4096/ga4gh-client.git@protobuf31#egg=ga4gh_client
```

Comment:
I may be missing the reasoning behind splitting the versions out of requirements and I can see the reasoning behind @dcolligan comment, but it seems to me that the best placeholder would be the versions required to build the current version of the server. However, the versions checked into master should probably reference official ga4gh repos. Then you can just reference the constraints file in requirements file and it should be buildable with a simple `pip install -r requirements.txt`.